PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/Endpoint/configuration_algorithm_classes/completion_construction_algorithms.cpp

```diff
@@ -0,0 +1,40 @@
+#include "completion_construction_algorithms.h"
+
+CompletionConstructionAlgorithms::CompletionConstructionAlgorithms(): dataToBeReadBits(0){}
+
+/**
+ * @brief Constructing a Completion TLP with Data Payload inside (return of a Configuration Read request)
+ * 
+ * @return TLP -> The Constructed TLP (CplD)
+ */
+TLP CompletionWithData::constructTLP()
+{
```

Comment:
I know, this will be implemented, but till then this will not be resolved as finished of course.

Reply:
Although, it's not finished, it can't be removed as it's called inside other functions and for the project to build, and it's right to hold this all branch for these three functions which are blocked on some other function in the Layers part.
so you can merge and it has another story on it's own, and will have a branch to implement it on.

Reply:
okay no problem if they will not make future problem in something else